### PR TITLE
XW-1928 | open search if input is focused before JS is loaded

### PR DESCRIPTION
**Pull request by Warkot**
Open search when JS is loaded if input is focused before JS is loaded.

Task: https://wikia-inc.atlassian.net/browse/XW-1928
Subtask: https://wikia-inc.atlassian.net/browse/XW-1984

@Wikia/x-wing

### CODE REVIEW

**Comment by rogatty** [comment]
LGTM

File: extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationSearch.js [hunk 1]
```diff
@@ -7,16 +7,14 @@ $(function ($) {
 		placeholderText = $searchInput.attr('placeholder'),
 		activeSearchClass = 'wds-search-is-active';
 
-	if ($searchInput.val().length === 0) {
```
Comment (Warkot) [inline extensions/wikia/DesignSystem/scripts/DesignSystemGlobalNavigationSearch.js:10]: I moved this down to add structure to the file:
- vars,
- function defs,
- listeners,
- code that runs upon script load.

**Comment by kvas-damian** [comment]
LGTM

**Comment by hakubo** [comment]
lgtm

**Comment by tomasznapieralski** [comment]
TMLG

**Pull request merged**